### DISCUSSION TITLE
Setup cloudlog in a shared host without allow_url_fopen ?

### DISCUSSION THREAD

**Discussion by alex9099** [magicbug/Cloudlog#2908] [Q&A] [answered]
Hey, I'm having some problems setting up cloudlog on a shared hosting without allow_url_fopen enabled, I've managed to do the initial configuration, but when updating the DXCC file it fails. 
Also, the popup to update shows the errors about allow_url_fopen being disabled.

So question is if it is possible to run cloudlog with allow_url_fopen = 0

**Comment by magicbug** [comment, marked as answer]
Currently at the moment you'll have issues if that's disabled

**Reply by alex9099** [reply]
I see, I'll talk to the hosting provider to see if they would enable it.
I've read some workarounds are possible in php, but not sure about details.

Perhaps some note would be nice to have on the installation notes about allow_url_fopen (altough the warnings are quite self explanatory)

Meanwhile, would it be possible to manually load DXCC?
